en/config/safety.md

```diff
@@ -251,11 +251,10 @@ The parameters that control when the quad-chute will trigger are listed in the t
-<a id="VT_FW_QC_HMAX"></a>[VT_FW_QC_HMAX](../advanced_config/parameter_reference.md#VT_FW_QC_HMAX) |Maximum height above the ground (if available, otherwise above home if available, otherwise above the local origin) where triggering a quad-chute is possible. At high altitudes there is a big risk to deplete the battery and therefore crash if quad-chuting there.
-<a id="VT_QC_HR_ERROR_I"></a>[VT_QC_HR_ERROR_I](../advanced_config/parameter_reference.md#VT_QC_HR_ERROR_I) | Quad-chute uncommanded descent threshold. This is the threshold for integrated height rate error to trigger a uncommanded-descent quad-chute. Only checked in altitude-controlled fixed-wing flight. Additional conditions that have to be met for uncommanded descent detection are a positive (climbing) height rate setpoint and a negative (sinking) current height rate estimate.
-<a id="VT_QC_T_ALT_LOSS"></a>[VT_QC_T_ALT_LOSS](../advanced_config/parameter_reference.md#VT_QC_T_ALT_LOSS) | Altitude loss threshold for quad-chute triggering during VTOL transition to fixed-wing
- flight. Quad-chute triggers the current altitude is more than this value below the altitude at the beginning of the transition.
+<a id="COM_QC_ACT"></a>[COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) | Quad-chute action after switching to multicopter flight. Can be set to: [Warning](#action_warning), [Return](#action_return), [Land](#action_land), [Hold](#action_hold).
+<a id="VT_FW_QC_HMAX"></a>[VT_FW_QC_HMAX](../advanced_config/parameter_reference.md#VT_FW_QC_HMAX) | Maximum quad-chute height, below which the quad-chute failsafe cannot trigger. This prevents high altitude quad-chute descent, which can drain the battery (and itself cause a crash). The height is relative to ground, home, or the local origin (in preference order, depending on what is available).
+<a id="VT_QC_ALT_LOSS"></a>[VT_QC_ALT_LOSS](../advanced_config/parameter_reference.md#VT_QC_ALT_LOSS) | Uncommanded descent quad-chute altitude threshold. An uncommanded descend quad-chute event will be triggered as soon as the vehicle drops below this value measured relative to the current altitude reference. The altitude reference is computed based on the vehicle altitude and the altitude setpoint as follows. If the vehicle flies above the current altitude sepoint, then the altitude reference is set to the current altitude setpoint. No event will be triggered as the altitude error is positive. If the vehicle flies below the current altitude setpoint, then the altitude reference is set to the maximum altitude the vehicle has achieved while flying below the current altitude setpoint. This is to make sure that an uncommanded quad-chute event is not triggered in case the altitude setpoint increases faster than the vehicle can follow. On the other hand an event will still be triggered if the vehicle continuously loses altitude.
```

Comment:
The discussion of implementation is more than a user needs. I think all they need to know is that you've thought about the fact that not being able to track when the altitude setpoint is increasing too fast, could look like an uncontrolled descent. So this bit:

> Note that the quad-chute is not triggered if the commanded altitude setpoint increases faster than the vehicle can follow (it is only triggered if the vehicle continuously loses altitude below the commanded setpoint). 

The question is, whether the "how this is done" information is useful to someone else, and whether or not we should keep it here, or somewhere else. 

How about something like this. It defines the terms, and uses a little puncutation (`<br>`) to improve the table layout.


```suggestion
<a id="VT_QC_ALT_LOSS"></a>[VT_QC_ALT_LOSS](../advanced_config/parameter_reference.md#VT_QC_ALT_LOSS) | Uncommanded descent quad-chute altitude threshold.<br><br>In altitude controlled modes, such as [Hold mode](../flight_modes/hold.html), [Position mode](../flight_modes/position_fw.md), [Altitude mode](../flight_modes/altitude_fw.md), or [Mission mode](../flight_modes/mission.md), a vehicle should track its current "commanded" altitude setpoint. The quad chute failsafe is triggered if the vehicle falls too far below the commanded setpoint (by the amount defined in this parameter).<br><br>Note that the quad-chute is is only triggered if the vehicle continuously loses altitude below the commanded setpoint; it is not triggered if the commanded altitude setpoint increases faster than the vehicle can follow.
```

Reply:
@hamishwillee Yep, that was exactly my thought as well and I was prepare to shorten it drastically. You've already done a great job here. Only one mistake you have "...is is...". Maybe that's how it's said in Australia though...